releasenotes/notes/target-based-workload-selection-a745168b07add397.yaml

```diff
@@ -0,0 +1,46 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
```

Comment:
This should be in the release notes of the cluster agent: https://github.com/DataDog/datadog-agent/tree/main/releasenotes-dca/notes

Reply:
Thank you!!! So sorry I missed this